IPs/RSKIP170.md

```diff
@@ -0,0 +1,91 @@
+# Peg-in to any address
+
+|RSKIP          |170           |
+| :------------ |:-------------|
+|**Title**      |Peg-in to any address |
+|**Created**    |01-SEP-20 |
+|**Author**     |MI |
+|**Purpose**    |USa |
+|**Layer**      |Core |
+|**Complexity** |2 |
+|**Status**     |Draft |
+
+## Abstract
+
+This RSKIP proposes a way for users to be able to lock funds from BTC, indicating the RSK address in which they want their RBTCs transferred to and a BTC refund address in case the peg-in can not be completed for some reason.
+
+## Motivation
+
+Allowing users to lock funds to any address in RSK, EOA or contract.
+
+## Specification
+
+This RSKIP proposes an extension to the current pegin workflow. By including an output with an OP_RETURN op code and certain payload in the transaction sent to the federation to perform the peg-in, the user can determine the RSK address where the RBTCs will be transferred to. 
+
+The BTC transaction sent to the federation must contain 
+ one output with value 0 and OP_RETURN op code followed by the following data:
+- `RSKT` (for RSK  transfer) as prefix indicating the output is for RSK (4) bytes.
+- Protocol version (2 bytes). Currently **01**
+- A valid RSK address (20 bytes)
+- [Optional] A valid BTC refund address in the following format:
+    - Address type (1 byte)
+        - 1 for P2PKH
+        - 2 for P2SH
+    - Hash needed to build refund address (20 bytes)
+        - For P2PKH type address: public key hash 
+        - For P2SH type address: script hash
+
+### Example output
+```
+{
+      "value": 0.00000000,
+      "n": 1,
+      "scriptPubKey": {
+        "asm": "OP_RETURN 52534b5400010e537aad84447a2c2a7590d5f2665ef5cf9b667a014f4c767a2d308eebb3f0f1247f9163c896e0b7d2",
+        "hex": "6a2b52534b5400010e537aad84447a2c2a7590d5f2665ef5cf9b667a014f4c767a2d308eebb3f0f1247f9163c896e0b7d2",
+        "type": "nulldata"
+      }
+    }
+```
+Payload included: `52534b5400010e537aad84447a2c2a7590d5f2665ef5cf9b667a014f4c767a2d308eebb3f0f1247f9163c896e0b7d2`
+
+- First 4 bytes correspond to the prefix that indicates the output has peg-in instructions for RSK (`RSKT` in hexa): `52534b54` 
```

Comment:
reading this it seems that what you are doing is getting the ASCII values of `RSKT`. If that's the idea specify it above please